### PR TITLE
fix(linkprops): update link props

### PR DESCRIPTION
**Pull request by AlasdairSwan**
BREAKING CHANGE move to link arrays and remove strings from component (and thus the need for i18n).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "@commitlint/config-angular": "^6.0.2",
     "@commitlint/prompt": "^6.0.2",
     "@commitlint/prompt-cli": "^6.0.2",
+    "@edx/edx-bootstrap": "^2.2.1",
```
Comment (AlasdairSwan) [inline package.json]: NB: this is necessary to run the footer dev server.

**Comment by douglashall** [comment]
So I'm assuming it didn't "just work" with the `@edx/frontend-i18n` requirement here?

**Comment by AlasdairSwan** [comment]
Sorry @douglashall I didn't get around to checking that. In prospectus we need to be able to update the link urls to point to Spanish versions of the pages so we would need this update for that anyway. I also think it makes sense to move all strings to passed in props, as opposed to some passed in and some hard-coded.

File: src/index.jsx [hunk 2]
```diff
@@ -61,42 +127,57 @@ const socialLinks = [
...
+    languageForm={{
+      activeLanguage: 'en',
+      screenReaderLabel: 'Choose Language',
+      submitLabel: 'Apply',
+      icon: <FontAwesomeIcon icon={faLanguage} size="2x" className="text-primary" />,
+      onLanguageSelected: () => {},
+    }}
+    copyright="© 2012–2019 edX Inc."
+    trademark={(
+      <React.Fragment>EdX, Open edX, and MicroMasters are registered trademarks of edX Inc. | 深圳市恒宇博科技有限公司 <a href="http://www.beian.miit.gov.cn">粤ICP备17044299号-2</a></React.Fragment>
```
Comment (adamstankiewicz) [inline src/index.jsx]: Personal preference but the shorthand for fragments is great if you have seen it before:

```jsx
<>
  <div />
  <div />
</>
```

**Pull request merged**